docs/customization.md

```diff
@@ -187,13 +187,21 @@ The default is to generate *everything* supported by the specific library. Once
 
 To control the specific files being generated, you can pass a CSV list of what you want:
 ```sh
+# generate the User and Pet APIs only
+--global-property apis="User:Pet"
+
 # generate the User and Pet models only
 --global-property models="User:Pet"
 
 # generate the User model and the supportingFile `StringUtil.java`:
 --global-property models=User,supportingFiles=StringUtil.java
 ```
 
+The names of the apis are the _sanitized_ tags converted to _UpperCamelCase_ and are generated by the following steps:
```

Comment:
shall we mention it's not always upper camel case and it really depends on the generators (e.g. python uses snake case) ?

Reply:
I will have to dig into this since I am still missing a piece somewhere apparently. Will try to take a look again this weekend.